Runtime/Native/Base/NativeClientBase.cs

```diff
@@ -58,9 +58,11 @@ internal NativeClientBase(BacktraceConfiguration configuration, BacktraceBreadcr
             _configuration = configuration;
             _breadcrumbs = breadcrumbs;
             _shouldLogAnrsInBreadcrumbs = ShouldStoreAnrBreadcrumbs();
+#if UNITY_ANDROID || UNITY_IOS || UNITY_STANDALONE_WIN
```

Comment:
Should all ANR code live in a new file guarded by these #ifdefs?

Reply:
yes, this is an option @rqbacktrace . We can put everything in the #ifdef and maybe it's a better idea because if we add a native client in the future we would need to keep in mind about anr watchdog thing.